src/lib.rs

```diff
@@ -1,26 +1,34 @@
 use wasm_bindgen::prelude::*;
+use tracing::{info, span, Level, instrument};
 
 mod api;
 
 // Called when the wasm module is instantiated
 #[wasm_bindgen(start)]
 fn main() -> Result<(), JsValue> {
+    let span = span!(Level::INFO, "fn main");
+    let _guard = span.enter();
+
     // Use `web_sys`'s global `window` function to get a handle on the global
     // window object.
-    let window = web_sys::window().expect("no global `window` exists");
+    let window = dbg!(web_sys::window().expect("no global `window` exists"));
     let document = window.document().expect("should have a document on window");
     let body = document.body().expect("document should have a body");
 
+    // debug!(window.js_class, document.js_class, ?body);
+
     // Manufacture the element we're gonna append
     let val = document.create_element("p")?;
     val.set_inner_html("Hello from Rust!");
+    info!("Main fn called and Rust says hello!");
 
     body.append_child(&val)?;
 
     Ok(())
 }
 
 #[wasm_bindgen]
+#[instrument]
```

Comment:
```suggestion
#[instrument]
#[wasm_bindgen]
```

build fails unless in this order

<img width="1232" alt="Screenshot 2024-04-30 at 12 04 29 PM" src="https://github.com/KyleWMiller/RKP/assets/1791092/17d9ab0e-390c-4686-95eb-8ca819505a16">